bigc/__init__.py

```diff
@@ -1 +1 @@
-from .client import BigCommerceAPI
+from .api import BigCommerceAPI
```

Comment:
This is will be a pretty big breaking change for other projects since at least in my tests I have had to monkeypatch `bigc.client.*` to mock the underlying APIs properly.

Reply:
FWIW, I'd generally recommend monkeypatching the `BigCommerceAPI` instance instead of the underlying classes.

Reply:
I am pretty sure I tried it that way before but monkeypatching rules always seemed to be an esoteric exercise so maybe (highly likely) I was doing it wrong.

Reply:
Either way, the next release will probably be 0.2.0 to signify the potential breakages caused by this update